### PR TITLE
AWS connector onboarding and autoprovisioning 

### PR DESCRIPTION
**Pull request by aymanjarrousms**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/awsAutoProvisioningSettings.json [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      },
+      "put": {
```
Comment (KrisBash) [inline specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/awsAutoProvisioningSettings.json]: Why is the connection made at MG level, but settings configured at sub level? That seems random and confusing. Why not embed settings in the connection object, if that is what it is controlling? Which sub is this? If there are dozens in the MG, what criteria is used to choose the sub in this PUT?

**Review by KrisBash — CHANGES_REQUESTED** [review]
please see comments. It'd be optimal to set up a meeting with ARM api review to discuss.

**Comment by erich-wang** [comment]
@aymanjarrousms, is this PR still valid? If yes, please address comments from **KrisBash**

**Comment by aymanjarrousms** [comment]
There are changes in the API, I will close it

**Pull request closed without merging**